src/containers/shared/components/Transaction/AMMCreate/Simple.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../SimpleRow'
+import { TransactionSimpleProps } from '../types'
+import { formatTradingFee } from '../../../utils'
+import { Account } from '../../Account'
+import { Amount } from '../../Amount'
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { amount, amount2, ammAccountID, tradingFee } = data.instructions
+  const value = <Amount value={amount} />
```

Comment:
nit: I'd prefer if these components were in-line in the return - it'll make the code easier to read.

Reply:
Can you explain? I'm not sure what you mean.